core/transfer/src/main/java/org/eclipse/dataspaceconnector/transfer/core/transfer/TransferProcessManagerImpl.java

```diff
@@ -495,65 +462,36 @@ private void transitionToCompleted(TransferProcess process) {
         observable.invokeForEach(listener -> listener.completed(process));
     }
 
-    /**
-     * Performs consumer-side or provider side provisioning for a service.
-     * <br/>
-     * On a consumer, provisioning may entail setting up a data destination and supporting infrastructure. On a provider, provisioning is initiated when a request is received and
-     * map involve preprocessing data or other operations.
-     */
-    private int provisionInitialProcesses() {
-        var processes = transferProcessStore.nextForState(INITIAL.code(), batchSize);
-        for (TransferProcess process : processes) {
-            DataRequest dataRequest = process.getDataRequest();
-            ResourceManifest manifest;
-            if (process.getType() == CONSUMER) {
-                // if resources are managed by this connector, generate the manifest; otherwise create an empty one
-                manifest = dataRequest.isManagedResources() ? manifestGenerator.generateConsumerManifest(process) : ResourceManifest.Builder.newInstance().build();
+    private boolean processInitial(TransferProcess process) {
```

Comment:
I would put it on this method, because to the uninitiated it might not be intrinsically clear that `processInitial` is actually starting the provision process. I'm referring to this comment:
```java
    /**
     * Performs consumer-side or provider side provisioning for a service.
     * <br/>
     * On a consumer, provisioning may entail setting up a data destination and supporting infrastructure. On a provider, provisioning is initiated when a request is received and
     * map involve preprocessing data or other operations.
     */
```

Reply:
woa, I think I removed it by error, reintroduced.